lib/src/node-sass/render.test.ts

```diff
@@ -10,41 +10,73 @@ import {pathToFileURL} from 'url';
 import * as sandbox from '../../../spec/helpers/sandbox';
 import {expectEqualIgnoringWhitespace} from '../../../spec/helpers/utils';
 import {compileString} from '../compile';
-import {render, RenderError, RenderResult} from './render';
+import {render, RenderError, RenderOptions, RenderResult} from './render';
 
 describe('render', () => {
   const sassPathWithoutExtension = 'test';
   const sassPath = `${sassPathWithoutExtension}.scss`;
 
+  function expectRenderResult(
+    renderOptions: RenderOptions,
+    runExpectations: (result: RenderResult) => void
+  ) {
+    return new Promise<void>((resolve, reject) => {
+      render(renderOptions, (error, result) => {
+        if (error || result === undefined) {
+          reject('Expected successful render');
+          return;
+        }
+        runExpectations(result);
+        resolve();
+      });
+    });
+  }
+
+  function expectRenderError(
+    renderOptions: RenderOptions,
+    runExpectations: (error: RenderError) => void
+  ) {
+    return new Promise<void>((resolve, reject) => {
+      render(renderOptions, error => {
+        if (error === undefined) {
+          reject('Expected render error');
+          return;
+        }
+        runExpectations(error);
+        resolve();
+      });
+    });
+  }
+
   describe('input', () => {
-    it('renders a file from an absolute path', async done => {
-      await sandbox.run(async () => {
+    it('renders a file from an absolute path', () => {
+      return sandbox.run(async () => {
         await fs.writeFile(sassPath, 'a {b: c}');
-        await render({file: p.resolve(sassPath)}, (_, result) => {
-          expectEqualIgnoringWhitespace(result!.css.toString(), 'a {b: c;}');
+
+        return expectRenderResult({file: p.resolve(sassPath)}, result => {
+          expectEqualIgnoringWhitespace(result.css.toString(), 'a {b: c;}');
```

Comment:
If you're already in an async function, imo it's generally better to use `await` for the final statement rather than `return`. It makes it clear that the value wrapped in the promise is irrelevant, and it makes it less likely that you'll accidentally break it when refactoring (for example, if you wrap the `return` in a `try`/`catch` it won't catch asynchronous errors but an `await` will).